chokuretsu/guide/play-game.html

```diff
@@ -0,0 +1,420 @@
+                this game.</p>
+            <p>To do this, you'll need to homebrew your Wii U, use UWUVCI to convert the ROM into a WUP (Wii U title)
+                and then install it to your system using WUP Installer GX2.</p>
+            <b>Requirements:</b>
+            <ul>
+                <li>Wii U system with internet access</li>
+                <li>SD card with at least 4GB capacity</li>
+                <li>Windows PC to run UWUVCI</li>
+            </ul>
+            <h4>Part 1 - Homebrew your Wii U</h4>
+            <div class="notice-box"><i class="fa-solid fa-triangle-exclamation"></i>&nbsp;Please take caution following
+                these steps to avoid bricking your system.
+            </div>
+            <ol>
+                <li>Follow the steps <a href="https://wiiu.hacks.guide/#/" target="_blank">on this guide</a> to install
+                    a CFW (Custom Firmware) Wrapper, Tiramisu, onto your system from your SD card via an exploit in the
+                    Internet Browser.
+                </li>
+                <li>Continue <a href="https://wiiu.hacks.guide/#/tiramisu/finalizing-setup?id=additional-homebrew-apps"
+                                target="_blank">following the steps</a> to install the Homebrew App Store to your
+                    system.
+                </li>
+                <li>Boot into the Homebrew App Store through Tiramisu by launching into Tiramisu on the Health & Safety
+                    screen of your console when powering it on.
+                </li>
+                <li>On the Homebrew App Store, navigate to or search for <code>WUP Installer GX2</code> (it should be
+                    near the top). Download and install it to your system, following the on-screen instructions.
+                </li>
+                <li>Once successfully installed, power off your Wii U system and remove the SD card. Re-insert it into
+                    your computer.
+                </li>
+            </ol>
+
+            <h4>Part 2 - Make ROM into WUP with UWUVCI</h4>
+            <ol>
+                <li>Ensure your SD Card is back inserted into your computer</li>
+                <li>On your PC, download and install the latest release of <a
+                        href="https://github.com/stuff-by-3-random-dudes/UWUVCI-AIO-WPF/releases/" target="_blank">UWUVCI</a>.
+                    If you are prompted by Microsoft Defender/Windows StartScreen, click "More Details" and "Run
+                    Anyway". The program may take some time to download tools and data.
+                </li>
+                <li>When the program loads up, select the DS icon on the left-hand side of the screen.</li>
+                <li>Select the following options:
+                    <ol type="i">
+                        <li>On the "Base ROM" dropdown, choose "Mario Kart DS (US)" or "Mario Kart DS (EU)" (depending
```

Comment:
Any particular reason for specifying Mario Kart DS in particular? I did Picross and it worked just fine.